example/dj/apps/app/tests/data_processors.py

```diff
@@ -154,6 +154,22 @@ def test_atomic_add_and_delete_issues_with_reverse(self, number, issue_data, use
         self.assert_valid_JSON_response(resp)
         self.assert_equal(issues_before_count, Issue.objects.all().count())
 
+        user_data['created_issues'] = (self.get_issue_data(), self.get_issue_data(), self.get_issue_data())
+        resp = self.put('%s%s/' % (self.USER_API_URL, user_pk), data=self.serialize(user_data))
```

Comment:
ach jo trošku s křížkem po funuse :( potřeboval jsem to mergnout opravím d dalším pr